requirements/quality.txt

```diff
@@ -8,7 +8,8 @@
 -e git+https://github.com/edx/django-rest-framework.git@1ceda7c086fddffd1c440cc86856441bbf0bd9cb#egg=djangorestframework==3.6.3
 appdirs==1.4.3            # via fs
 argparse==1.4.0           # via caniusepython3
-astroid==1.5.3            # via edx-lint, pylint, pylint-celery
+astroid==1.6.5; python_version < "3.0"  # via edx-lint, pylint, pylint-celery
+astroid==2.1.0; python_version >= "3.0"  # via edx-lint, pylint, pylint-celery
```

Comment:
This file is auto-generated by running `make upgrade`, and the underlying `pip-tools` package we use to do that currently has a bug in handling multiple versions of the same package like this: https://github.com/jazzband/pip-tools/issues/518 .  So I agree in principle with this approach, but this will get blown away and break the next time we upgrade dependencies.